package.json

```diff
@@ -39,6 +39,7 @@
     "typescript": "^3.3.1"
   },
   "dependencies": {
+    "@types/cookie": "^0.3.3",
```

Comment:
Could you move this in devDependencies.

Reply:
That won't work, because when consumers install this module the `@types/cookie` package will not be installed, and thus no types would show.

On top of that, unless the consumer package (project) installs `@types/cookie` manually, then TS will throw compilation errors.

Reply:
Then let's embed those relative types. I'm -1 in adding this dependency.